--- origsrc/gstreamer-0.10.25/gst/Makefile.am	2009-09-16 16:00:08.000000000 -0500
+++ src/gstreamer-0.10.25/gst/Makefile.am	2009-11-01 23:13:30.269623400 -0600
@@ -250,7 +250,7 @@ Gst-@GST_MAJORMINOR@.gir: $(INTROSPECTIO
 		-I$(top_builddir) \
 		-DIN_GOBJECT_INTROSPECTION=1 \
 		--c-include='gst/gst.h' \
-		--library=gstreamer-0.10 \
+		--library=libgstreamer-0.10.la \
 		--include=GLib-2.0 \
 		--include=GObject-2.0 \
 		--include=GModule-2.0 \
--- origsrc/gstreamer-0.10.25/libs/gst/base/Makefile.am	2009-09-16 16:00:08.000000000 -0500
+++ src/gstreamer-0.10.25/libs/gst/base/Makefile.am	2009-11-01 23:12:17.152295900 -0600
@@ -58,7 +58,7 @@ GstBase-@GST_MAJORMINOR@.gir: $(INTROSPE
 		-I$(top_srcdir) \
 		$(gir_cincludes) \
 		--add-include-path=$(top_builddir)/gst \
-		--library=gstbase-0.10 \
+		--library=libgstbase-0.10.la \
 		--include=Gst-0.10 \
 		--libtool="$(top_builddir)/libtool" \
 		--pkg gstreamer-0.10 \
--- origsrc/gstreamer-0.10.25/libs/gst/check/Makefile.am	2009-10-01 09:36:41.000000000 -0500
+++ src/gstreamer-0.10.25/libs/gst/check/Makefile.am	2009-10-19 00:52:27.219536100 -0500
@@ -117,7 +117,7 @@ GstCheck-@GST_MAJORMINOR@.gir: $(INTROSP
 		-I$(top_srcdir) \
 		$(gir_cincludes) \
 		--add-include-path=$(top_builddir)/gst \
-		--library=gstcheck-0.10 \
+		--library=libgstcheck-0.10.la \
 		--include=Gst-0.10 \
 		--libtool="$(top_builddir)/libtool" \
 		--pkg gstreamer-0.10 \
--- origsrc/gstreamer-0.10.25/libs/gst/controller/Makefile.am	2009-09-16 16:00:08.000000000 -0500
+++ src/gstreamer-0.10.25/libs/gst/controller/Makefile.am	2009-11-01 23:12:24.390708500 -0600
@@ -47,7 +47,7 @@ GstController-@GST_MAJORMINOR@.gir: $(IN
 		-I$(top_srcdir) \
 		$(gir_cincludes) \
 		--add-include-path=$(top_builddir)/gst \
-		--library=gstcontroller-0.10 \
+		--library=libgstcontroller-0.10.la \
 		--include=Gst-0.10 \
 		--libtool="$(top_builddir)/libtool" \
 		--pkg gstreamer-0.10 \
--- origsrc/gstreamer-0.10.25/libs/gst/dataprotocol/Makefile.am	2009-09-16 16:00:08.000000000 -0500
+++ src/gstreamer-0.10.25/libs/gst/dataprotocol/Makefile.am	2009-11-01 23:12:29.569917500 -0600
@@ -37,7 +37,7 @@ GstDataprotocol-@GST_MAJORMINOR@.gir: $(
 		-I$(top_srcdir) \
 		$(gir_cincludes) \
 		--add-include-path=$(top_builddir)/gst \
-		--library=gstdataprotocol-0.10 \
+		--library=libgstdataprotocol-0.10.la \
 		--include=Gst-0.10 \
 		--libtool="$(top_builddir)/libtool" \
 		--pkg gstreamer-0.10 \
--- origsrc/gstreamer-0.10.25/libs/gst/net/Makefile.am	2009-09-16 16:00:08.000000000 -0500
+++ src/gstreamer-0.10.25/libs/gst/net/Makefile.am	2009-11-01 23:12:35.357527600 -0600
@@ -38,7 +38,7 @@ GstNet-@GST_MAJORMINOR@.gir: $(INTROSPEC
 		-I$(top_srcdir) \
 		$(gir_cincludes) \
 		--add-include-path=$(top_builddir)/gst \
-		--library=gstnet-0.10 \
+		--library=libgstnet-0.10.la \
 		--include=Gst-0.10 \
 		--libtool="$(top_builddir)/libtool" \
 		--pkg gstreamer-0.10 \
