--- origsrc/gstreamer-0.10.20/gst/gstplugin.c	2008-01-21 11:08:07.000000000 -0600
+++ src/gstreamer-0.10.20/gst/gstplugin.c	2008-06-18 16:47:07.484375000 -0500
@@ -77,7 +77,7 @@
 /* static variables for segfault handling of plugin loading */
 static char *_gst_plugin_fault_handler_filename = NULL;
 
-#ifndef HAVE_WIN32
+#ifdef HAVE_SIGACTION
 static gboolean _gst_plugin_fault_handler_is_setup = FALSE;
 #endif
 
